app/api/user/user.go

```diff
@@ -18,4 +18,5 @@ func (srv *Service) SetRoutes(router chi.Router) {
 	router.Use(render.SetContentType(render.ContentTypeJSON))
 	router.Use(auth.UserIDMiddleware(&srv.Config.Auth))
 	router.Get("/user/invitations", service.AppHandler(srv.getInvitations).ServeHTTP)
+	router.Get("/user/memberships", service.AppHandler(srv.getMemberships).ServeHTTP)
```

Comment:
As it's not just the memberships of users in general but of the authenticated one, shouldn't we do `/user/me/memberships` (which would allow `/user/{user_id}/memberships` later as well)

Reply:
I meant 'user'='me' otherwise it would be 'users'. I can replace 'user' by 'me' if you will.

Reply:
'/users/{user_id}/' is possible, but '/user/{user_id}' is not

Reply:
I like 'user' better than 'me'

Reply:
Do we really need to have '/users/{user_id}/memberships'?

Reply:
Fixed. It will be /current-user/memberships